### DISCUSSION TITLE
Nested contrasts for 3 factors

### DISCUSSION THREAD

**Discussion by tatiana-pashkova** [RePsychLing/SMLP2023#33] [Q&A] [answered]
Today we talked about nested contrasts that will test a contrast in factor F2 nested under the levels of factor F1 (the model formula y ~ 1 + F1/F2). What would be the best way to do the same thing but nested inside the combination of levels of factors F1 and F3?

For example, I have 3 variables: F1 Formality (formal/informal), F2 Bilingualism (biling/monoling) and F3 Mode (spoken/written), Bilingualism (biling/monoling)
I want to know the difference between monolinguals and bilinguals within formal spoken, formal written, informal spoken, and informal written. 

One way to do this, as I imagine, would be to create a factor F4 with 4 levels (formal spoken, formal written, informal spoken, and informal written), and fit a model y ~ 1 + F4/F2. Is this an acceptable approach? Is there any other approach?

**Comment by tatiana-pashkova** [comment, marked as answer]
The answer that Reinhold gave in person in class was that you can either do y ~ 1 + F4/F2 as suggested in the question or do y ~ 1 + F1/F3/F2 (F2 nested under F3 and nested under F1)

**Reply by kliegl** [reply]
`F1/F3/F2` works in R. In Julia you need to write `F1/(F3/F2)`.  I include two examples at the end of the `mrk17` script.  There is a more elaborate R/lme4-based version  in the git-repo for SLMP2020 in script `mrk17_contrasts.Rmd`:
https://github.com/smlp2020-stream4/SMLP2020/tree/master/scripts .  Also lots of other demos. 

I still need to convert this to Julia MixedModels.
